src/schema/v2/artwork/index.ts

```diff
@@ -335,6 +335,27 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return false
         },
       },
+      canRequestLotConditionsReport: {
+        type: GraphQLBoolean,
+        description:
+          "Can a user request a lot conditions report for this artwork?",
+        resolve: ({ sale_ids }, _options, { saleLoader }) => {
+          if (sale_ids && sale_ids.length > 0) {
+            const sale_id = sale_ids[0]
```

Comment:
Out of 2.7M artworks in the database, there were only 49 artworks that have multiple sales, none of which was created in 2018 or 2019. It is probably safe to assume that these were assigned to multiple sales unintentionally. Even if it was possible (e.g. https://www.artsy.net/artwork/hurvin-anderson-sun-shade-to-add), we seem to look at [the first `sale`](https://github.com/artsy/metaphysics/blob/73d40d98f13dab5b2fe9f0f2d6742fe081d351b4/src/schema/v2/artwork/index.ts#L639-L643) for the `sale` field on `Sale`, and checking against all the associated sales would lead to a situation where the check looks at the flag for an already closed sale, which we don't want.

Reply:
> we seem to look at the first sale for the sale field on Sale, and checking against all the associated sales would lead to a situation where the check looks at the flag for an already closed sale, which we don't want.

Makes perfect sense – I didn't catch the fact that we already consider the first sale to be main one – 👍